packages/stencil/src/components/datacom-list/datacom-li.css

```diff
@@ -15,7 +15,8 @@
   font-weight: 400;
 }
 
-li::marker {
+.dc-li-item::marker {
```

Comment:
All the CSS here does nothing against the marker pseudo element. (Double check it for yourself if I'm wrong.)

You should remove this selector and CSS, unless you have a specific reason you need this.